main.nf

```diff
@@ -109,11 +117,10 @@ include { BWA_MEM   } from './modules/nf-core/software/bwa/mem/main'   addParams
 /* --           RUN MAIN WORKFLOW              -- */
 ////////////////////////////////////////////////////
 
-// Info required for completion email and summary
+// Info required for completion email and summary   
 def multiqc_report = []
 
 workflow {
```

Comment:
As mentioned in the previous review, and I see Harshil also commented it now on the #bactmap channel, all this part inside the `workflow {}` can be moved into a new file called `bactmap.nf` in the main pipeline directory. That leaves the `main.nf` with only the boilerplate code